autoconfigure/README.md

```diff
@@ -0,0 +1,36 @@
+# Configuration
+
+## Storage configurations
+
+| Configuration | Description | Default |
+|---------------|-------------|---------|
+| `KAFKA_STORE_SPAN_CONSUMER_ENABLED` | Enable partitioning of incoming span batches into individual spans keyed by trace ID. | `true` |
```

Comment:
I think there are too many variables here.. usually we put the most important as environemnt variables and then leave the rest to spring boot options ex `--zipkin.storage.kafka.consumer-enbled=false`

for example, this distracts if conflated with things you can already do with search enabled false.. also I wouldn't recommend promoting flags that would break functionality. in other words, lets remove all of these flags until someone needs to split hairs. this makes it simpler and lets you re-use docs around search enabled etc.

Reply:
you're right, I have removed `aggregation-enabled` as it is tricky to start properly with the lazy initialization, and I will start it as when `consumer-enabled=true`. Also replaced `store-enabled` with `search-enabled` to reuse existing config.